### DISCUSSION TITLE
What's new in this feedback ?

### DISCUSSION THREAD

**Discussion by mspronesti** [community/community#13452] [Other Feature Feedback, Questions, & Ideas]
Perhaps I don't follow this many people to notice it, but I don't see many differences with respect to the previous feed. I only see what my followed and followers fork, star or create, plus some repos suggestion here and there (which was already done by github on the right side of the web application).

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
